tests/Application/Actions/Hooks/StripeChargeUpdateTest.php

```diff
@@ -452,7 +516,15 @@ public function testUnsupportedRefundAmount(): void
 
         $body = $this->getStripeHookMock('ch_unsupported_partial_refund');
         $donation = $this->getTestDonation();
-        $webhookSecret = $container->get('settings')['stripe']['accountWebhookSecret'];
+        /**
+         * @var array{
+         *    stripe: array{
+         *      accountWebhookSecret: string
```

Comment:
The need to repeat this and similar makes me think it might be worth changing the value of 'settings' in the container from an array to an object with an actual class. Could be read-only with all public properties. Not a blocker here, but might be worth doing before repeating this sort of thing too many times. Using `@var` is error-prone, there's no check that the var actually hold what we say it holds.

Reply:
@bdsl I think we talked about this a couple of weeks ago. The latest Slim Skeleton and newer apps do this so we have a model to follow, but it's not a super quick change.